daffodil-core/src/main/scala/org/apache/daffodil/dsom/Term.scala

```diff
@@ -746,7 +741,7 @@ trait Term
    *
    * What stops this is when the end of an enclosing element has to be next.
    */
-  final def possibleNextChildElementsInInfoset: Seq[ElementBase] = LV('possibleNextChildElementsInInfoset) {
+  def possibleNextChildElementsInInfoset: Seq[ElementBase] = LV('possibleNextChildElementsInInfoset) {
```

Comment:
Any particular reason final was removed? It doesn't look like this patch changes anything to override this function.

Reply:
I think this was an oversight. I'd changed it to a val during debugging, and forgot the "final" during restoration